src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml

```diff
@@ -92,7 +92,7 @@
 
     <property>
       <name>yarn.nodemanager.hostname</name>
-      <value>{{ host_config['ip'] }}</value>
+      <value>{POD_IP}</value>
```

Comment:
Though, it's good for me. But you'd better decouple it from thhis PR,

Reply:
En, Ideally we need to separate it from this PR.
Put it here because I want to avoid a complex fixing of the start script.